functions/go/inflate-helm-chart/README.md

```diff
@@ -0,0 +1,238 @@
+function with `kpt fn eval`, the flag `--network` must be used for remote charts,
+and the flag `--mount` must be used for local charts. See the examples for inflating
+[local] and [remote] charts.
+
+It can be used declaratively when run with kustomize. To run the function with kustomize,
+the `network` field is needed for remote charts and the `mounts` field is needed for local charts.
+
+### FunctionConfig
+
+<!--mdtogo:Long-->
+
+There are 2 kinds of `functionConfig` supported by this function:
+
+- `ConfigMap`
+- A custom resource of kind `InflateHelmChart`
+
+#### `ConfigMap`
+To use a `ConfigMap` as the `functionConfig`, the desired parameters must be
+specified in the `data` field:
+
+```yaml
+data:
+  chartHome: string
+  configHome: string
+  name: string
+  version: string
+  repo: string
+  releaseName: string
+  namespace: string
+  valuesFile: string
+  includeCRDs: string
+```
+
+
+| Field        |  Description | Example
+| -----------: |  ----------- | -----------
+`chartHome`    | A filepath to a directory of charts. The function will look for the chart in this local directory before attempting to pull the chart from a specified repo. Defaults to "tmp/charts". When run in a container, this path MUST have the prefix "tmp/". | tmp/charts
+`configHome`   | (exec only) Defines a value that the function should pass to helm via the HELM_CONFIG_HOME environment variable. If omitted, {tmpDir}/helm is used, where {tmpDir} is some temporary directory created by the function for the benefit of helm. | /tmp/helm/config
+`name`         | The name of the chart | minecraft
+`version`      | The version of the chart | 3.1.3
+`repo`         | A URL locating the chart on the internet | https://itzg.github.io/minecraft-server-charts
+`releaseName`  | Replaces RELEASE_NAME in the chart template output | test
+`namespace`    | Sets the target namespace for a release (`.Release.Namespace` in the template) | my-namespace
+`valuesFile`   | valuesFile is a remote or local file path to a values file to use instead of the default values that accompanied the chart. The default values are in '{chartHome}/{name}/values.yaml', where `chartHome` and `name` are the parameters defined above. | tmp/values.yaml <br> <br> https://raw.githubusercontent.com/config-sync-examples/helm-components/main/cert-manager-values.yaml
+`includeCRDs`  | Specifies if Helm should also generate CustomResourceDefinitions, defaults to 'false' | true
+
+
+The only required field is `name`.
+
+#### `InflateHelmChart`
+A `functionConfig` of kind `InflateHelmChart` has the following supported parameters: 
+
+```yaml
+helmGlobals:
+  chartHome: string
+  configHome: string
+helmCharts:
+- name: string
+  version: string
+  repo: string
+  releaseName: string
+  namespace: string
+  valuesInline: map[string]interface{}
+  valuesFile: string
+  valuesMerge: string
+  includeCRDs: bool
+```
+
+| Field        |  Description | Example
+| -----------: |  ----------- | -----------
```

Comment:
Some comments above also apply here. Please audit thie table.

Reply:
I changed both tables according to your comments